README.md

```diff
@@ -88,6 +88,14 @@ Persistence Query usage example to obtain a stream with all events tagged with "
 
     val queries = PersistenceQuery(system).readJournalFor[CassandraReadJournal](CassandraReadJournal.Identifier)
     queries.eventsByTag("someTag", Offset.noOffset)
+    
+## Cleanup of tag_views table
+
+By default the tag_views table keeps tagged events indefinitely.  Depending on the volume of events
```

Comment:
, also when the original events have been removed.

Reply:
+1, adding that